config/config.go

```diff
@@ -132,13 +132,23 @@ func GetOptionalParameters(config *Config, defaultLidarDataRateMsec, defaultIMUD
 			optionalConfigParams.ImuName = imuName
 			strMovementSensorDataFreqHz, ok := config.MovementSensor["data_frequency_hz"]
 			if !ok {
-				logger.Debugf("config did not provide movement_sensor[data_frequency_hz], setting to default value of %d", 1000/defaultIMUDataRateMsec)
+				if optionalConfigParams.LidarDataRateMsec == 0 {
+					optionalConfigParams.ImuDataRateMsec = 0
+					logger.Warn("camera[data_frequency_hz] is set to 0, " +
+						"setting movement_sensor[data_frequency_hz] to 0")
+				} else {
+					optionalConfigParams.ImuDataRateMsec = defaultIMUDataRateMsec
```

Comment:
Should this be `1000 / defaultIMUDataRateMsec`?

Reply:
No - we are passing the default in as a data rate, so no conversion needs to happen

Reply:
can we fix the log on the next line in that case